### PR TITLE
Update minimum CMake version to 3.19

### PR DESCRIPTION
**Pull request by ntadej**
Update minimum CMake version to 3.19. Will need a separate PR to update to a container that is containing this CMake version but the one we have now should do it by then.

/cc @alanchenboy

### CODE REVIEW

**Review by louwers — CHANGES_REQUESTED** [review]
Looks like the Node.js workflow also needs to be updated.

**Comment by ntadej** [comment]
> Looks like the Node.js workflow also needs to be updated.

Added a setup-cmake acction, but not sure if it will work on a self-hosted runner. Let's see.

**Comment by louwers** [comment]
@ntadej Looks like it is not platform aware (self-hosted runner uses ARM64).

File: .github/actions/qt5-build/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/maplibre/linux-builder:centos7-cmake3.10
+FROM ghcr.io/maplibre/linux-builder:centos7-latest
```
Comment (ntadej) [inline .github/actions/qt5-build/Dockerfile:1]:
```suggestion
FROM ghcr.io/maplibre/linux-builder:centos7-cmake3.19
```

**Comment by alanchenboy** [comment]
Thanks for this updating

**Comment by ntadej** [comment]
@louwers, I think we need to just figure out how to update CMake in the Linux ARM runner? Can you just do it system-wide maybe?

**Comment by louwers** [comment]
@ntadej Thanks for the heads up. Yes, I can do that.

**Review by louwers — APPROVED** [review]

**Pull request merged**